data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1667 @@
+	description "Bring the <bunks> ranchers to their world, <destination>."
+	passengers 47
+	landing
+	source "Chosen Nexus"
+	destination "Shifting Sand"
+	to offer
+		has "Resistance: Evacuation 3: done"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch License: done"
+	to fail
+		has "Heliarch Investigation 2: active"
+	on offer
+		conversation
+			`The ranchers arrive to the hangars shortly after you land, most of them Arachi, with a few Saryds and Kimek in their midst. They rush into your ship, all <bunks> of them present.`
+			branch resistance
+				has "Resistance: Questions: done"
+			`	"All ready to leave, we are," one Arach tells you as most of the others head to find their bunks.`
+			choice
+				`	"Got it, we'll leave soon."`
+					goto accept
+				`	"How did getting that loan go?"`
+			`	"What?" She looks at you with puzzled eyes, until another Arach lightly whacks one of her legs. "Ah, yes, the loan. Well enough. Agreed to grant us the amount needed, they have." She looks back at the other Arach, who stares at her for a moment. "Well, very tired we all are, Captain, so head to my bunk, I will."`
+				goto accept
+			label resistance
+			`	"Thought, I had, that left behind we would be," one Arach tells you after you've closed the hatch. "Hurry to <planet>, you should. Risk an 'inspection' by the Heliarchs, we cannot."`
+			choice
+				`	"Got it, we'll leave soon."`
+					goto accept
+				`	"Is everyone here?"`
+			`	"Thankfully, yes. Lost nobody this time, we have," she says. "Less brutish on their searches on worlds like <origin>, the Heliarchs are. Managed to avoid them well enough, we did. Or, at least, hope so I do."`
+			label accept
+			`	They all move to their bunks, and you set a course for <destination>.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort with the <bunks> ranchers hasn't arrived in the system yet. Better depart and wait for it.`
+	on complete
+		payment 748300
+		conversation
+			`The Arachi, along with the scattered few Saryds and Kimek, rush out of your ship when you land, thanking you quickly as each one leaves. The Arachi who seemed to be leading the group give you more of a farewell, chatting a bit, until they see Elirom approaching. They grunt, and leave with the rest. Elirom watches them leave after handing you your payment of <payment>.`
+			branch resistance
+				has "Resistance: Questions: done"
+			choice
+				`	"They didn't seem very happy to see you."`
+					goto happy1
+				`	"How come you're here? I thought you hired me because you didn't have any ships available."`
+			`	"Hmm? Ah, well returned to where we had met, one of the ships did. Made it in time to pick up the ranchers it could not, though, so very grateful for your service, the company is once again," he explains. "Brought here by that ships captain I was,, for wished I did to apologize to the ranchers in person. Afraid I am, that hear my explanations they will not."`
```

Comment:
```suggestion
			`	"Hmm? Ah, well returned to where we had met, one of the ships did. Made it in time to pick up the ranchers it could not, though, so very grateful for your service, the company is once again," he explains. "Brought here by that ship's captain I was, for wished I did to apologize to the ranchers in person. Afraid I am, that hear my explanations they will not."`
```